src/lib/account-profile/account-profile-menu/account-profile-menu.component.html

```diff
@@ -0,0 +1,15 @@
+<td-menu class="account-profile-menu">
+  <!--header-->
+  <mat-list td-menu-header>
+    <mat-list-item *ngIf="name" (click)="_blockEvent($event)">
```

Comment:
What if no name but yes email?

Reply:
accounted for this in my latest update. will make email top line style now if name is missing